### PR TITLE
i18n(es): Translate `prerequisites/index.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
Translated `prerequisites/index.mdx` to spanish 🇪🇸

### CODE REVIEW

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+i18nReady: true
+---
+
+import { Tabs, TabItem, Card } from '@astrojs/starlight/components';
+
+Para empezar a construir tu proyecto con Tauri primero deberás instalar algunas dependencias:
+
+1. [Dependencias del Sistema](#dependencias-del-sistema)
+2. [Rust](#rust)
+3. [Configurar para Objetivos Móviles](#configurar-para-objetivos-móviles) (solo se requiere si se desarrolla para móviles)
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]: `Objetivos` sounds incredible weird in this context, I would just say

```suggestion
3. [Configurar para Móvil](#configurar-para-móvil) (solo se requiere si se desarrolla para móviles)
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+## Dependencias del Sistema
+
+Sigue el enlace para comenzar para tu sistema operativo respectivo:
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
Sigue el enlace para comenzar según tu sistema operativo:
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+  </TabItem>
+  <TabItem label="NixOS">
+
+TODO: Need to build out NixOS instructions
+
+  </TabItem>
+</Tabs>
+
+Si tu distribución no está incluida arriba, es posible que desees consultar [Awesome Tauri en GitHub](https://github.com/tauri-apps/awesome-tauri#guides) para ver si se ha creado una guía.
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
Si tu distribución no está incluida arriba, puedes consultar [Awesome Tauri en GitHub](https://github.com/tauri-apps/awesome-tauri#guides) para ver si se ha creado una guía.
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 4]
```diff
@@ -0,0 +1,315 @@
...
+
+Siguiente: [Instalar Rust](#rust)
+
+### macOS
+
+Tauri usa [Xcode](https://developer.apple.com/xcode/resources/) varias dependencias de desarrollo de macOS e iOS.
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
Tauri usa [Xcode](https://developer.apple.com/xcode/resources/) y varias dependencias de desarrollo de macOS e iOS.
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 5]
```diff
@@ -0,0 +1,315 @@
...
+
+Descarga e instala Xcode desde uno de los siguientes lugares:
+
+- [App Store de Mac](https://apps.apple.com/gb/app/xcode/id497799835?mt=12)
+- [Sitio web del Desarrollador de Apple](https://developer.apple.com/xcode/resources/).
+
+Asegúrate de iniciar Xcode después de instalarlo para que pueda terminar de configurarse.
+
+<details>
+<summary>¿Solo desarrollo para objetivos de escritorio?</summary>
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
<summary>¿Solo estás desarollando para escritorio?</summary>
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 6]
```diff
@@ -0,0 +1,315 @@
...
+  <TabItem label="Windows">
+  
+  Visita https://www.rust-lang.org/tools/install para instalar `rustup`.
+
+  </TabItem>
+</Tabs>
+
+Asegúrate de reiniciar tu Terminal (y en algunos casos tu sistema) para que los cambios surtan efecto.
+
+Siguiente: [Configurar para Objetivos Móviles](#configurar-para-objetivos-móviles) si deseas construir para Android e iOS. De lo contrario, [Crea un Proyecto](/2/guide/create/).
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
Siguiente: [Configurar para Móvil](#configurar-para-móvil) si deseas construir para Android e iOS. De lo contrario, [Crea un Proyecto](/2/guide/create/).
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+
+## Configurar para Objetivos Móviles
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
## Configurar para Móvil
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 8]
```diff
@@ -0,0 +1,315 @@
...
+
+Si deseas apuntar tu aplicación para Android o iOS, entonces hay algunas dependencias adicionales que necesitas instalar:
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
Si deseas compilar tu aplicación para Android o iOS, entonces hay algunas dependencias adicionales que necesitas instalar:
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 9]
```diff
@@ -0,0 +1,315 @@
...
+```ps
+[System.Environment]::SetEnvironmentVariable("ANDROID_HOME", "$env:LocalAppData\Android\Sdk", "User")
+[System.Environment]::SetEnvironmentVariable("NDK_HOME", "$env:LocalAppData\Android\Sdk\ndk\25.0.8775105", "User")
+```
+
+</TabItem>
+
+</Tabs>
+
+5. Agrega los objetivos de Android con `rustup`:
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
5. Agrega los destinos de compilación de Android con `rustup`:
```

File: src/content/docs/es/2/guide/prerequisites/index.mdx [hunk 10]
```diff
@@ -0,0 +1,315 @@
...
+
+Siguiente: [Configurar para iOS](#ios) o [Crear un proyecto](/2/guide/create/).
+
+### iOS
+
+:::caution[Solo macOS]
+El desarrollo de iOS requiere Xcode y solo está disponible en macOS.
+:::
+
+Agrega los objetivos de iOS con `rustup` en Terminal:
```
Comment (marc2332) [inline src/content/docs/es/2/guide/prerequisites/index.mdx]:
```suggestion
Agrega los destinos de compilación de iOS con `rustup` en Terminal:
```

**Review by marc2332 — CHANGES_REQUESTED** [review]
just some suggestions, nice work!! 💯

**Comment by dreyfus92** [comment]
Thank you so much for the review @marc2332 😄

**Review by marc2332 — APPROVED** [review]

**Review by lorenzolewis — APPROVED** [review]
Absolutely amazing, thank you @dreyfus92 and thanks @marc2332 for the thorough review 💪

**Pull request merged**